### PR TITLE
Add TS 1.5 Blob Type

### PR DESCRIPTION
**Pull request by alexmoore**
Please ensure the following is present in your Pull Request:

- [X] Unit tests for your change
- [X] Integration tests (if applicable)

Pull Requests that are small and limited in scope are most welcome.

Thank you!

### CODE REVIEW

**Comment by alexmoore** [comment]
Also please "Squash and merge" when merging.

File: test/unit/ts/data.js [hunk 1]
```diff
@@ -75,15 +82,16 @@ for (var i = 0; i < rows.length; i++) {
                 break;
             case TS.ColumnType.Timestamp:
                 if (val) {
-                    cell.setTimestampValue(Long.fromNumber(val.getTime()));
```
Comment (lukebakken) [inline test/unit/ts/data.js]: Why was this changed?

**Comment by lukebakken** [comment]
👍 Once Travis CI passes, :shipit:
